lap3dkernel/c++SIMD/VCL_AgnerFog_version1/vectorf256.h

```diff
@@ -1785,6 +1785,11 @@ static inline Vec4d sqrt(Vec4d const & a) {
     return _mm256_sqrt_pd(a);
 }
 
+// aprrox 1/sqrt(a) function, takes double type as function argument

```

Comment:
VCL version 1 or version 2 don't have simd approx rsqrt for double type argument, so I added to the local VCL lib.